CMakeLists.txt

```diff
@@ -14,7 +14,8 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
   "${CMAKE_CURRENT_LIST_DIR}/cmake/CeleritasMakeRulesOverride.cmake")
 
 project(Celeritas VERSION "${Celeritas_VERSION}" LANGUAGES CXX)
-cmake_policy(VERSION 3.18...3.28)
+cmake_policy(VERSION 3.18...3.31)
```

Comment:
It seems like this is redundant with the `cmake_minimum_required` above... maybe delete the cmake policy here? (I somehow thought it had to be after `project`)

Reply:
The build succeeds with CMP0175 set to OLD, but fails with NEW.  I will try to fix it...